packages/build/src/plugins_core/dev_blobs_upload/index.ts

```diff
@@ -0,0 +1,18 @@
+import { uploadBlobs } from '../blobs_upload/index.js'
+import { type CoreStep, type CoreStepCondition } from '../types.js'
+
+const condition: CoreStepCondition = async (...args) => {
+  const {
+    constants: { IS_LOCAL },
+  } = args[0]
+  return IS_LOCAL && ((await uploadBlobs.condition?.(...args)) ?? true)
```

Comment:
I don't think `IS_LOCAL` is a particularly useful check given that I wouldn't expect us to run `dev` anywhere but locally, but it doesn't seem to hurt to perform this check, I guess? I can remove it if you agree it's not useful.